UPGRADE-3.0.md

```diff
@@ -1011,3 +1011,7 @@ UPGRADE FROM 2.x to 3.0
 ### Swiftmailer Bridge
 
  * `Symfony\Bridge\Swiftmailer\DataCollector\MessageDataCollector` was removed. Use the `Symfony\Bundle\SwiftmailerBundle\DataCollector\MessageDataCollector` class instead.
+
+### HttpFoundation
+
+* `Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface` don't implement anymore `\IteratorAggregate` interface
```

Comment:
Might be also good to suggest what should be done instead. For example:

> `Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface` no longer implements the `IteratorAggregate` interface. Use the `all()` method instead of iterating over the flash bag.